src/modules/dashboard/components/ColonyHome/ColonySubscription/ColonySubscription.tsx

```diff
@@ -68,6 +74,8 @@ const ColonySubscription = ({
 
   const isNetworkAllowed = checkIfNetworkIsAllowed(networkId);
 
+  const colonyMenu = isSubscribed ? styles.colonySubscribed : '';
```

Comment:
It's a nitpick but I think it's more readable and clear when it's inline for className...